RELEASE_NOTES.rst

```diff
@@ -21,6 +21,276 @@
+- Fix Scheduler in daemon mode doesn't create PID at the specified location (#38117)
+- Properly serialize TaskInstancePydantic and DagRunPydantic (#37855)
+- Fix graph task state border color (#38084)
+- Add back methods removed in security manager (#37997)
+- Don't log "403" from worker serve-logs as "Unknown error". (#37933)
+- Fix execution data validation error in ``/get_logs_with_metadata`` endpoint (#37756)
+- Fix task duration selection (#37630)
+- Fix construct ``docker.TLSConfig`` for ``docker>=7`` (#37481)
+- Refrain from passing ``encoding`` to the SQL engine in SQLAlchemy v2 (#37545)
+- Fix 'implicitly coercing SELECT object to scalar subquery' in latest dag run statement (#37505)
+- Clean up typing with max_execution_date query builder (#36958)
+- Optimize max_execution_date query in single dag case (#33242)
+- Make ``hachicorp`` dependencies consistent between ``provider.yaml`` and ``pyproject.toml`` (#36758)
+- Fix list dags command for get_dagmodel is None (#36739)
+- Load ``consuming_dags`` attr eagerly before dataset listener (#36247)
+
+
+Miscellaneous
+"""""""""""""
+- Lift the limit on importlib_resources library (#38294)
+- Bump minimum version of ``blinker`` add where it requires (#38140)
+- Bump follow-redirects from 1.15.4 to 1.15.6 in /airflow/www (#38156)
+- Bump Cryptography to > 39.0.0 (#38112)
+- Add Python 3.12 support (#36755, #38025, #36595)
+- Avoid use of ``assert`` outside of the tests (#37718)
+- Update ObjectStoragePath for universal_pathlib>=v0.2.2 (#37930)
+- Resolve G004: Logging statement uses f-string (#37873)
+- Switch from --user to venv for PROD image and enable uv (#37796)
+- Upgrade google-ads version (#37787)
+- Update build and install dependencies. (#37910)
+- Resolve G003: "Logging statement uses +" (#37848)
+- Bump sanitize-html from 2.11.0 to 2.12.1 in /airflow/www (#37833)
+- Fix "Exception statement has no effect" (#37722)
+- Update to latest installer versions. (#37754)
+- Deprecate smtp configs in airflow settings / local_settings (#37711)
+- Add smtp provider to prod image (#37739)
+- Deprecate PY* constants into the airflow module (#37575)
+- Remove usage of deprecated ``flask._request_ctx_stack`` (#37522)
+- Remove redundant ``login`` attribute in ``airflow.__init__.py`` (#37565)
+- Upgrade to FAB 4.3.11 (#37233)
+- Remove SCHEDULED_DEPS which is no longer used anywhere since 2.0.0 (#37140)
+- Replace ``datetime.datetime.utcnow`` by ``airflow.utils.timezone.utcnow`` in core (#35448)
+- Bump aiohttp min version to avoid CVE-2024-23829 and CVE-2024-23334 (#37110)
+- Upgrade yarn stack in pre-commits (#37048)
+- Generate Python client in reproducible way (#36763)
+- Move config related to FAB auth manager to FAB provider (#36232)
+- Remove MSSQL support form Airflow core (#36514)
+- Modify SmtpNotifier to accept template with defaults (#36226)
+- Upgrade to just released ``pip`` 23.3.2 (#36271)
+- Remove ``daskexcutor`` provider from Airflow code (#36269)
```

Comment:
I think in core's context, this is a docs change?

Reply:
I think we should leave it at misc since there are some code elements, though they are removals.

Reply:
Hmm, I don't see any. Which changes are you referring to?